src/ROUTES.js

```diff
@@ -42,6 +42,10 @@ export default {
     getSettingsAddLoginRoute: type => `settings/addlogin/${type}`,
     SETTINGS_PAYMENTS_TRANSFER_BALANCE: 'settings/payments/transfer-balance',
     SETTINGS_PAYMENTS_CHOOSE_TRANSFER_ACCOUNT: 'settings/payments/choose-transfer-account',
+    SETTINGS_PERSONAL_DETAILS: 'settings/profile/personal-details',
+    SETTINGS_PERSONAL_DETAILS_LEGAL_NAME: 'settings/profile/personal-details/legal-name',
+    SETTINGS_PERSONAL_DETAILS_DATE_OF_BIRTH: 'settings/profile/personal-details/date-of-birth',
+    SETTINGS_PERSONAL_DETAILS_ADDRESS: 'settings/profile/personal-details/address',
```

Comment:
```suggestion
    SETTINGS_PERSONAL_DETAILS_LEGAL_NAME: `${SETTINGS_PERSONAL_DETAILS}/legal-name`,
    SETTINGS_PERSONAL_DETAILS_DATE_OF_BIRTH: `${SETTINGS_PERSONAL_DETAILS}/date-of-birth`,
    SETTINGS_PERSONAL_DETAILS_ADDRESS: `${SETTINGS_PERSONAL_DETAILS}/address`,
```

Reply:
hadn't seen us doing this before! i like it!

Reply:
We've been doing this for IOU and other routes 😄